manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/mockapis/OauthApi.groovy

```diff
@@ -296,7 +296,7 @@ class OauthApi extends WireMockRule {
 
     void stubAuthCreateUser() {
         this.stubFor(
-                put(urlPathMatching("/auth/api/authuser/.*"))
+                post(urlPathMatching("/auth/api/authuser/.*"))
```

Comment:
this would make more sense to be `/auth/api/authuser/create`

Reply:
Fixed